mediapipe/calculators/ovms/modelapiovmsadapter.hpp

```diff
@@ -26,13 +26,15 @@
 #include <adapters/inference_adapter.h>  // model_api/model_api/cpp/adapters/include/adapters/inference_adapter.h
 #include <openvino/openvino.hpp>
 
+#include "ovms.h"  // NOLINT
```

Comment:
Why do we need this header in .hpp?

Reply:
Yes we needed to move the method to header for visibility of symbols on windows, same as we did for ovms code in some cases.